run_kind_ci.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+KIND_CI_PROJECTS=("sriov-cni" "sriov-ib" "antrea" "ipoib" "network-operator" "antrea")
+KIND_CI_PHASES=("deploy-kind" "utilities" "deploy-project" "test" "undeploy-project" "undeploy-kind")
+# SUPPORTED_PHASES is temporally used until all phases are supported
+SUPPORTED_PHASES=("deploy-kind" "undeploy-kind")
+export PHASES_TO_RUN=("${SUPPORTED_PHASES[@]}")
+
+usage() {
+  echo "usage: run_kind_ci.sh [[--project <name>] [--kind-config <conf-fil>] [--num-workers <num>]"
+  echo "                       [--skip-phases <phases>] [--kubeconfig <path>]  [-h]]"
+  echo ""
+  echo "--project              Project to run CI: ${KIND_CI_PROJECTS[*]}"
+  echo "                       Required field"
+  echo "--kind-config          Kind configuration file, if provided skip rendering related parameters"
+  echo "                       if provided, ignores rendering related parameters (num-workers)"
+  echo "--num-workers          Number of worker nodes. DEFAULT: 2 worker"
+  echo "--skip-phases          Comma separated phases, phases: ${KIND_CI_PHASES[*]}"
+  echo "--kubeconfig           KUBECONFIG for kind cluster"
+  echo ""
+}
+
+containsElement() {
+  local value="$1"
+  local list="$2"
+  shift
+  for e in $list; do
+    [[ "$e" == "$value" ]] && return 0
+  done
+  return 1
+}
+
+parse_args() {
+  while [ "$1" != "" ]; do
+    case $1 in
+    --project)
+      shift
+      project=$1
+      containsElement "$project" "${KIND_CI_PROJECTS[*]}"
+      if [[ "$?" != "0" ]]; then
+        echo "Unknown project ci $project"
+        usage
+        exit 1
+      fi
+      export PROJECT=$project
+      ;;
+    --num-workers)
+      shift
+      if ! [[ "$1" =~ ^[0-9]+$ ]]; then
+        echo "Invalid num-workers: $1"
+        usage
+        exit 1
+      fi
+      export KIND_NUM_WORKER=$1
+      ;;
+    --kind-config)
+      shift
+      kind_conf=$1
+      if test ! -f "$kind_conf"; then
+        echo "$kind_conf does not  exist"
+        usage
+        exit 1
+      fi
+      export KIND_CONFIG=$1
+      ;;
+    --skip-phases)
+      shift
+      SKIP_PHASES=$1
+      IFS="," read -a SKIP_PHASES <<<"$1"
+      for phase in "${SKIP_PHASES[@]}"; do
+        containsElement "$phase" "${KIND_CI_PHASES[*]}"
+        if [[ "$?" != "0" ]]; then
+          echo "Unknown phase $phase"
+          usage
+          exit 1
+        fi
+      done
+
+      PHASES_TO_RUN=()
+      for phase in "${SUPPORTED_PHASES[@]}"; do
+        containsElement "$phase" "${SKIP_PHASES[*]}"
+        if [[ "$?" == "1" ]]; then
+          PHASES_TO_RUN+=("$phase")
+        fi
+      done
```

Comment:
i think it would be a good idea to have this in a function. maybe define the functions before reading inputs.